specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/dataVersion.json

```diff
@@ -0,0 +1,387 @@
+      "delete": {
+        "tags": [
+          "DataVersion"
+        ],
+        "x-ms-examples": {
+          "Delete DataVersion": {
+            "$ref": "./examples/deleteDataVersion.json"
+          }
+        },
+        "description": "Delete Data Version entity.",
+        "operationId": "DataVersions_Delete",
+        "parameters": [
+          {
+            "$ref": "machineLearningServices.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/DataVersionNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DataVersionVersionsParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "machineLearningServices.json#/definitions/MachineLearningServiceError"
+            }
+          },
+          "200": {
+            "description": "Success"
+          },
+          "204": {
+            "description": "No Content"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "DataVersion"
+        ],
+        "x-ms-examples": {
+          "Get DataVersion": {
+            "$ref": "./examples/getDataVersion.json"
+          }
+        },
+        "description": "Get Data Version container.",
+        "operationId": "DataVersions_Get",
+        "parameters": [
+          {
+            "$ref": "machineLearningServices.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "machineLearningServices.json#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/DataVersionNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DataVersionVersionsParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DataVersionResource"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "machineLearningServices.json#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      },
+      "put": {
```

Comment:
Is this (or any of these operations) an LRO / Async Operation?

Reply:
yes they are async

Reply:
than make sure you mark any such operations with the x-ms-long-running-operation extension: https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation